script/src/syscalls/load_header.rs

```diff
@@ -114,16 +114,14 @@ impl<'a, DL: DataLoader + 'a> LoadHeader<'a, DL> {
         header: &HeaderView,
     ) -> Result<(u8, u64), VMError> {
         let field = HeaderField::parse_from_u64(machine.registers()[A5].to_u64())?;
-        let epoch = match self.data_loader.get_block_epoch(&header.hash()) {
-            Some(epoch) => epoch,
-            None => return Ok((ITEM_MISSING, 0)),
-        };
+        let epoch = header.epoch();
 
         let result = match field {
             HeaderField::EpochNumber => (SUCCESS, store_u64(machine, epoch.number())?),
-            HeaderField::EpochStartBlockNumber => {
-                (SUCCESS, store_u64(machine, epoch.start_number())?)
-            }
+            HeaderField::EpochStartBlockNumber => (
+                SUCCESS,
+                store_u64(machine, header.number() - epoch.index())?,
```

Comment:
I suggest using `overflowing_sub` here, even those data are verified in `ckb`, but syscalls may be used out of ckb and also more friendly for mock return an error rather than panic.

Reply:
Resolved, returns a `VMError::Unexpected` here like other syscall, please review again, thanks.